app/views/gobierto_people/people/person_events/index.html.erb

```diff
@@ -66,6 +66,15 @@
 
 <% description([title, t("gobierto_people.layouts.application.title"), @site.title].compact.join('. ')) %>
 
+<% content_for :gobierto_footer_content do %>
+  <%= javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.30.1/moment.min.js' %>
+  <%= javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/fullcalendar/3.10.2/fullcalendar.min.js' %>
```

Comment:
Why is the CDN necessary?

Reply:
LOL https://github.com/PopulateTools/gobierto/pull/4267#issuecomment-2459554480

Reply:
right 🤔